### DISCUSSION TITLE
How to solve the blank box appearing at the intersection of scrollbars

### DISCUSSION THREAD

**Discussion by piterzhou** [eclipse-theia/theia#13842] [Q&A]
When both horizontal and vertical scrollbars appear simultaneously, a white box appears at the intersection，How to solve it,thanks

**Comment by JonasHelming** [comment]
Hi,

could you provide more details, please? In which view does this occur, which Theia version and which operating system. Also could you please create a screenshot.

Best regards,
Jonas
